src/Controller/DefaultController.php

```diff
@@ -162,11 +159,18 @@ public function showSiteMapXml(EntityManagerInterface $entityManager, FundingOrg
 
         $datasets = $entityManager->getRepository(Dataset::class)->findBy($criteria);
 
-        $response = new StreamedResponse(function () use ($datasets) {
+        $sitemapMinDateParam = $this->getParameter('sitemap_min_date');
+        if (!is_string($sitemapMinDateParam)) {
+            throw new \InvalidArgumentException('The parameter "sitemap_min_date" must be a valid string.');
```

Comment:
Don't throw any exceptions, we want the sitemap to still work if this string is not there, just don't use it in that case. Leave it `null` or adjust your twig logic.